frontend/src/core/comments/components/AddComment.js

```diff
@@ -0,0 +1,90 @@
+/* @flow */
+
+import * as React from 'react';
+import { Localized } from '@fluent/react';
+
+import './Comment.css';
+
+import { UserAvatar } from 'core/user'
+
+type Props = {|
+    user: string,
+    username: string,
+    imageURL: string,
+|};
+
+
+export default function AddComments(props: Props) {
+    const { user, username, imageURL } = props;
+
+    if (!user) {
+        return null;
+    }
+
+    const submitComment = (event: SyntheticMouseEvent<HTMLButtonElement>) => {
+        event.preventDefault();
+        let comment=document.querySelector('#comment-input').value
+        document.getElementById('comment-input').value = ''
+        console.log(comment);
+    };
+
+    const rejectWithComment = (event: SyntheticMouseEvent<HTMLButtonElement>) => {
+        event.preventDefault();
+        console.log('Reject');
+    };
+
+    const approveWithComment = (event: SyntheticMouseEvent<HTMLButtonElement>) => {
+        event.preventDefault();
+        console.log('Approve');
+    };
+
+    return <div className='comment add-comment'>
+        <UserAvatar
+            user={ user }
+            username={ username }
+            title=''
+            imageUrl={ imageURL }
+        />
+        <form className='container'>
+            <div className='content'>
```

Comment:
I don't think we need this `<div>`. It should be easier to style `<input>` directly.